ShelterModule/Services/Implementations/Announcements/AnnouncementCommand.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Database;
+using Microsoft.EntityFrameworkCore;
+using ShelterModule.Models.Announcements;
+using ShelterModule.Services.Interfaces.Announcements;
+
+namespace ShelterModule.Services.Implementations.Announcements;
+
+public class AnnouncementCommand : IAnnouncementCommand
+{
+    private readonly PetShareDbContext _dbContext;
+
+    public AnnouncementCommand(PetShareDbContext dbContext)
+    {
+        _dbContext = dbContext;
+    }
+
+    public async Task<Announcement> AddAsync(Announcement announcement, CancellationToken token = default)
+    {
+        var entityAnnouncement = announcement.ToEntity();
+        _dbContext.Add(entityAnnouncement);
```

Comment:
We should add this to a specific set instead (so: `_dbContext.Announcements.Add(...)`)